src/test/java/com/google/api/codegen/testdata/discoveries/cloudmonitoring.v2beta2.json.overrides

```diff
@@ -1,13 +1,7 @@
 {
   "java": {
     "methods": {
-      "cloudmonitoring.metricDescriptors.list": {
-        "requestBodyType": null
-      },
-      "cloudmonitoring.timeseries.list": {
-        "requestBodyType": null
-      },
-      "cloudmonitoring.timeseriesDescriptors.list": {
+      ".*list": {
```

Comment:
I considered it, but thought it more likely such a hypothetical method would follow their existing pattern. Seems like compile check would catch it either way.